src/anders_mathiesen_ex/ex05/walker_sim.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Anders Mathiesen'
+__email__ = 'andermat@nmbu.no'
+
+
+import random
+
+
+class Walker:
+    """
+     Simulate a student’s way home to Pentagon after a hard night at Samfunnet
+    """
+    def __init__(self, x0, home):
+        self.home = home
+        self.x = x0
+        self.x0 = x0
+        self.steps = 0
+
+    def move(self):
+        """
+        a method move to take one step
+        """
+        self.steps += 1
+        if random.randint(0, 1):
```

Comment:
not so cool. Here you are evaluating `(0,1)` as true/false. It would be nicer to use some kind of random bool, `random.choice()` or do the following: `self.position += random.randint(0, 1) * 2 - 1`